101-bundle-json.md

```diff
@@ -26,7 +26,7 @@ A `bundle.json` is broken down into the following categories of information:
 - A list of outputs (name, type and location) that the application produces
 - A set of schema definitions used to validate user input
 
-The `bundle.json` is located at `/cnab/bundle.json` in the invocation image's runtime filesystem.
+The bundle definition is [made accessible](103-bundle-runtime.md#bundle-definition) from inside the invocation image in order to allow the run tool to reference information in the file. The bundle.json MUST be mounted to /cnab/bundle.json.
```

Comment:
@trishankatdatadog thanks for updating this section. The link is redundant since the same information is present in both places. What are your thoughts on either dropping the link or keeping the original wording and adding the link?

Reply:
I can remove the link